app/src/main/res/layout/fragment_exceptions_domains.xml

```diff
@@ -38,8 +39,9 @@
         <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/exceptionList"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical" />
+            android:layout_height="0dp"
+            android:orientation="vertical"
+            android:layout_weight="0.9"/>
```

Comment:
Is there any specific reason why we are changing the `RecyclerView` layout configurations ? :)

Reply:
If we are having many exceptions "remove exceptions" buttons is pushed out of the screen

Reply:
Thanks for the clarification 👍🏽